EasyReflectometry/sample/elements/layers/layer_apm.py

```diff
@@ -219,53 +226,65 @@ def from_pars(
             interface=interface,
         )
 
+    @property
+    def area_per_molecule(self) -> Parameter:
+        """
+        :return: Area per molecule
+        """
+        return self._area_per_molecule
+
     @property
     def solvent(self) -> Material:
-        """Getter for the solvent material."""
-        return self.material.material_b
+        """
+        :return: Solvent material
+        """
+        return self.material.solvent
 
     @solvent.setter
     def solvent(self, new_solvent: Material) -> None:
         """
         :param new_solvent: New solvent material.
         """
-        self.material.material_b = new_solvent
+        self.material.solvent = new_solvent
 
     @property
     def solvation(self) -> Parameter:
-        """Getter for the solvation parameter."""
```

Comment:
I'm not sure `solvation` is the right name for a parameter. `Solvation` is a process.
Maybe `solvation_parameter` or a variation thereof?

Reply:
I have looked a bit further into how `solvation` is used in the code and the name seems quite off.

To my understanding we use `solvation` to describe the fraction of a layer (solvent and chemical formula) that is not covered by the chemical (molecules).

This seems to be a large change and I have created an issue to handle this: https://github.com/easyScience/EasyReflectometryLib/issues/101